guides/common/modules/ref_migrating-from-foreman_webhooks.adoc

```diff
@@ -0,0 +1,14 @@
+[id="migrating-webhooks_{context}"]
+= Migration to Foreman Webhooks
+
+The legacy `foreman_hooks` provides full access to model objects that the webhooks plugin does not intentionally provide.
+
+The scope of what is available is limited by the safemode and all objects and macros are both subject to API stability promise and are fully documented.
+
+The number of events triggered by webhooks is substantially less than with `foreman_hooks`.
+
+{Project} are designed to be safe.
```

Comment:
Could you please rephrase this? I am unsure what's the purpose of this statement.